src/main/java/by/exadel/internship/entity/Location.java

```diff
@@ -0,0 +1,18 @@
+package by.exadel.internship.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Location {
```

Comment:
Remove please the class from this PR. It'll be added in separate PR and with all appropriate annotation.

Reply:
done